docs/tutorials/google/advanced_calibration.ipynb

```diff
@@ -0,0 +1,799 @@
+{
```

Comment:
<p>yeah definitely use sampler. Also you should pass project_id to get_engine (or get_engine_sampler) since you go through the effort of allowing it to be input as a variable.</p><p> </p><p>I wish we had some indication of progress or visual indication that something is happening when you run this cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>